sound/soc/intel/boards/sof_sdw.c

```diff
@@ -201,6 +201,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_RT715_DAI_ID_FIX |
 					SOF_SDW_PCH_DMIC),
 	},
+	{
+                .callback = sof_sdw_quirk_cb,
+                .matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Google_Brya0"),
```

Comment:
Hmm, the "Brya0" seems a bit odd. Have we used these numbers before in the family names? If it's in the coreboot, maybe unavoidable, but just pops out in the definition.

Reply:
Yes its from coreboot But we haven't used Product family as match, will change it to Name match.
Thanks